updater/scripts/git-lfs-download.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# Calculate Git LFS traffic per day in bytes
+#
+
+echo "lfs traffic [B]"
+zcat -f /var/log/haproxy.log.1* |
+    LANG=C perl -ne '/\s200\s(\d+)\s-\s\-\s----\s.*GET (?:\/storage)?\/lfs\/\d+\// && {$traffic += $1}; END {print $traffic}'
```

Comment:
Why is an optional `/storage` prefix allowed? I couldn’t find an occurrence of that in our logs. Is this for compatbility with another version of GitHub Enterprise?

Reply:
For the record: According to GitHub Enterprise support, the `/storage` prefix is introduced with GitHub Enterprise 2.13. As a measure of compatibility, the `/storage` prefix is optionally matched hence.